### PR TITLE
HTTPS support in ELB

### PR DESCRIPTION
**Pull request by giorgiosironi**
ELB support a certificate reference that points to our wildcard certificate being safely stored into AWS IAM.

Moreover, we configure other interesting parameters of the ELB like sticky sessions and the connection timeout, using them for `journal--end2end` which is currently being recreated.

### CODE REVIEW

File: projects/elife.yaml [hunk 1]
```diff
@@ -59,6 +59,12 @@ defaults:
             # external hdd
             size: 10 # GB
             device: /dev/sdh
+        elb:
+            # elb defaults only used if an 'elb' section present in project
+            stickiness: False
+            protocol: http
+            idle_timeout: 60
+            certificate: ASCAIRXYIRFBOR5QSDP5M
```
Comment (lsh-0) [inline projects/elife.yaml]: is this some kind of ID?

**Comment by lsh-0** [comment]
I'm happy for this to be merged so long as that mysterious value I commented on isn't sensitive

File: projects/elife.yaml (hunk 1 above)
Comment (lsh-0) [inline projects/elife.yaml]: perhaps give this a comment in the defaults?

`# id for ... ll: ABCDEFGHIJ123ABC5M`

**Comment by giorgiosironi** [comment]
Incidentally, I had to replace the mysterious reference with its ARN for the ELB to correctly recognize it; so now it's much more readable.

**Comment by lsh-0** [comment]
I was wondering about why the ARN wasn't being used. I'm pleased it is now.

Ready to merge?

**Comment by giorgiosironi** [comment]
Almost, I am still missing testing cookie stickiness and have to rebuild the stack one more time.

**Comment by giorgiosironi** [comment]
Added not only stickiness, but also buildvars updates to make them work on multiple nodes. When we deploy, we must change the buildvars on all EC2 instances and then run the update as normal. The `buildvars.*` tasks now act on all nodes too.

**Pull request merged**